dependencies/pip/requirements.in

```diff
@@ -13,7 +13,7 @@ https://bitbucket.org/fomcl/savreaderwriter/downloads/savReaderWriter-3.3.0.zip#
 -e git+https://github.com/kobotoolbox/ssrf-protect@9b97d3f0fd8f737a38dd7a6b64efeffc03ab3cdd#egg=ssrf_protect
 
 # Regular PyPI packages
-Django>=2.2,<2.3
+Django==3.2
```

Comment:
Don't use exact version of Django, use >=,< syntax

Reply:
:facepalm:  I thought I already changed it when you made me change it in KPI. 
Done!